docs/basics/userguide/plot_stats.md

```diff
@@ -0,0 +1,7999 @@
+    INFO:rlberry_logger:[Sb3-PPO[worker: 7]] | max_global_step = 10240 | time/iterations = 4 | rollout/ep_rew_mean = 44.64 | rollout/ep_len_mean = 44.64 | time/fps = 115 | time/time_elapsed = 70 | time/total_timesteps = 8192 | train/learning_rate = 0.0003 | train/entropy_loss = -0.633265808224678 | train/policy_gradient_loss = -0.0230875588960771 | train/value_loss = 50.796407270431516 | train/approx_kl = 0.010280366986989975 | train/clip_fraction = 0.1154296875 | train/loss = 19.713285446166992 | train/explained_variance = 0.26903200149536133 | train/n_updates = 30 | train/clip_range = 0.2 |
+    [38;21m[INFO] 14:46: [Sb3-PPO[worker: 6]] | max_global_step = 10240 | time/iterations = 4 | rollout/ep_rew_mean = 44.09 | rollout/ep_len_mean = 44.09 | time/fps = 114 | time/time_elapsed = 71 | time/total_timesteps = 8192 | train/learning_rate = 0.0003 | train/entropy_loss = -0.635085660405457 | train/policy_gradient_loss = -0.019524750619893894 | train/value_loss = 51.484919738769534 | train/approx_kl = 0.009435750544071198 | train/clip_fraction = 0.088720703125 | train/loss = 15.843541145324707 | train/explained_variance = 0.25749021768569946 | train/n_updates = 30 | train/clip_range = 0.2 |  [0m
+    INFO:rlberry_logger:[Sb3-PPO[worker: 6]] | max_global_step = 10240 | time/iterations = 4 | rollout/ep_rew_mean = 44.09 | rollout/ep_len_mean = 44.09 | time/fps = 114 | time/time_elapsed = 71 | time/total_timesteps = 8192 | train/learning_rate = 0.0003 | train/entropy_loss = -0.635085660405457 | train/policy_gradient_loss = -0.019524750619893894 | train/value_loss = 51.484919738769534 | train/approx_kl = 0.009435750544071198 | train/clip_fraction = 0.088720703125 | train/loss = 15.843541145324707 | train/explained_variance = 0.25749021768569946 | train/n_updates = 30 | train/clip_range = 0.2 |
+    [38;21m[INFO] 14:46: [Sb3-PPO[worker: 9]] | max_global_step = 10240 | time/iterations = 4 | rollout/ep_rew_mean = 45.69 | rollout/ep_len_mean = 45.69 | time/fps = 117 | time/time_elapsed = 69 | time/total_timesteps = 8192 | train/learning_rate = 0.0003 | train/entropy_loss = -0.6443713787943125 | train/policy_gradient_loss = -0.012846545978391077 | train/value_loss = 55.9754634976387 | train/approx_kl = 0.008374381810426712 | train/clip_fraction = 0.0560546875 | train/loss = 18.558977127075195 | train/explained_variance = 0.21047407388687134 | train/n_updates = 30 | train/clip_range = 0.2 |  [0m
+    INFO:rlberry_logger:[Sb3-PPO[worker: 9]] | max_global_step = 10240 | time/iterations = 4 | rollout/ep_rew_mean = 45.69 | rollout/ep_len_mean = 45.69 | time/fps = 117 | time/time_elapsed = 69 | time/total_timesteps = 8192 | 
```

Comment:
Please do not include colab stuffs. They are not useful for the doc and may cause bugs.

Reply:
Don't know how to remove, this was generated automatically with a jupyter to markdown converter.

Reply:
You can just remove them manually.